net-analyzer/hexinject/hexinject-1.5.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DESCRIPTION="Hexadecimal and raw packet injector and sniffer. Can be easily
```

Comment:
Strictly speaking, I think the limit of 80 chars includes `DESCRIPTION=""` part. And don't end description with a dot if it's not a proper sentence.